src/client/components/componentWithSliders/componentWithSliders.js

```diff
@@ -1,19 +1,41 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './componentWithSliders.css';
 import PresetsDropdownSelector from '../presetsDropdownSelector';
 import SavePresetsButton from '../savePresetsButton';
 import AllBlocks from './allSliders';
+import SavePressetModalBlock from './savePressetModalBlock';
 
-const ComponentWithSliders = () => (
-  <section className="SlidersComponent">
-    <header className="SlidersComponent__header">
-      <SavePresetsButton />
-      <PresetsDropdownSelector />
-    </header>
-    <main className="SlidersComponent__main">
-      <AllBlocks />
-    </main>
-  </section>
-);
+
+class ComponentWithSliders extends Component {
+  state = {
+    isModalBlockShow: false,
+  }
+
+  showHideModalBlock = (clickOrEscKeyEvent) => {
```

Comment:
I would suggest to rename this parameter, to minimize your if statement, for eg. mouseEvent, or smth like that

Reply:
renamed